src/main/java/org/logstash/dissect/search/DelimiterSearchStrategy.java

```diff
@@ -0,0 +1,5 @@
+package org.logstash.dissect.search;
+
+public interface DelimiterSearchStrategy {
```

Comment:
Ok, that's good. Still, if you're parsing text in Java you typically operate on characters instead of bytes.